### PR TITLE
stabilize branding

### PR DESCRIPTION
**Pull request by nagilson**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
     <VersionPrefix>8.0.300</VersionPrefix>
     <!-- When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
```
Comment (joeloff) [inline eng/Versions.props:9]: What's the reason for changing the default? This would produce stable versions for local builds when you don't set this.

Comment (nagilson) [reply eng/Versions.props:9]: This was supposed to be 8 0 3, not main. Whoops. Thank you for catching it!

**Review by marcpopMSFT — APPROVED** [review]

**Comment by nagilson** [comment]
rerunning osx failed jobs again..

**Pull request merged**